src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      data-qa="big"
+      action="http://localhost:8080/"
+      method="post"
+    >
+      <label class="label">
+        <input
+          class="search"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form
+      class="form"
+      action="http://localhost:8080/"
+      method="post"
+    >
+      <label class="label label-small">
+        <input
+          data-qa="small"
```

Comment:
data-qa attribute should be on form instead input
![Screenshot from 2023-07-28 10-52-15](https://github.com/yarkob/layout_search-bar-airbnb/assets/33029864/82e45565-c0a4-4a77-8035-756361fdc371)